pep-0723.rst

```diff
@@ -684,3 +770,11 @@ Copyright
 
 This document is placed in the public domain or under the
 CC0-1.0-Universal license, whichever is more permissive.
+
+.. [1] A large number of users use scripts that are version controlled. For
+   example, `the SREs that were mentioned <723-comment-block_>`_ or
+   projects that require special maintenance like the
+   `AWS CLI <https://github.com/aws/aws-cli/tree/4393dcdf044a5275000c9c193d1933c07a08fdf1/scripts>`__
+   or `Calibre <https://github.com/kovidgoyal/calibre/tree/master/setup>`__.
+.. [2] For example, projects like Hatch and Poetry have their own backends
+   and may wish to support this use case only when their backend is used.
```

Comment:
These should go in a new Footnotes section, below References but above Copyright.

Reply:
Oops good catch, done!